table/tables/tables.go

```diff
@@ -1190,23 +1194,167 @@ func init() {
 	table.MockTableFromMeta = MockTableFromMeta
 }
 
+// sequenceCommon cache the sequence value.
+// `alter sequence` will invalidate the cached range.
+// `setval` will recompute the start position of cached value.
+type sequenceCommon struct {
+	meta *model.SequenceInfo
+	// base < end when increment > 0.
+	// base > end when increment < 0.
+	end  int64
+	base int64
```

Comment:
How about naming them `low` and `high`? **`high` is always greater than `low`.**

Reply:
not readlly, if use low/high, that will mean low < high when increment > 0, but **low > high** when increment < 0, which is weird.